frappe/custom/doctype/custom_field/test_custom_field.py

```diff
@@ -4,13 +4,13 @@
 import frappe
 from frappe.tests.utils import FrappeTestCase
 
+from .custom_field import create_custom_fields
```

Comment:
Let's avoid relative imports for consistency.

Reply:
Updated